internal/cloud/realm/dependencies_test.go

```diff
@@ -97,7 +105,7 @@ func TestRealmDependencies(t *testing.T) {
 	})
 }
 
-func parseZipArchive(t *testing.T, file *os.File) map[string]string {
+func getZipFileNames(t *testing.T, file *os.File) map[string]bool {
```

Comment:
[meganit] use return value of `map[string]struct{}` instead (it may be _slightly_ more performant, but it also conveys there's no use for the values of the map here, we're simply keeping track of the key set)